test/hotspot/gtest/runtime/test_committed_virtualmemory.cpp

```diff
@@ -91,7 +91,7 @@ class CommittedVirtualMemoryTest {
   static void test_committed_region_impl(size_t num_pages, size_t touch_pages, int* page_num) {
     const size_t page_sz = os::vm_page_size();
     const size_t size = num_pages * page_sz;
-    char* base = os::reserve_memory(size, !ExecMem, mtThreadStack);
+    char* base = os::reserve_memory(size, mtThreadStack, !ExecMem);
```

Comment:
```suggestion
    char* base = os::reserve_memory(size, mtThreadStack);
```

Reply:
ExecMem is a parameter, which could be false, so that would make it `true` for `reserve_memory`, so need to keep it?